README.rst

```diff
@@ -182,6 +186,57 @@ Basic usage
     nodes = Node.objects.with_tree_fields().without_tree_fields()
 
 
+Understanding tree fields
+-------------------------
+
+When using ``with_tree_fields()``, each node gets three additional attributes:
+
+- **``tree_depth``**: An integer representing the depth of the node in the tree
+  (root nodes have depth 0)
+- **``tree_path``**: An array containing the primary keys of all ancestors plus
+  the current node itself, representing the path from root to current node
+- **``tree_ordering``**: An array containing the ordering/ranking values used
+  for sibling ordering at each level of the tree hierarchy
+
+The key difference between ``tree_path`` and ``tree_ordering``:
+
+.. code-block:: python
+
+    # Example tree structure:
+    #   Root (pk=1, order=0)
+    #   ├── Child A (pk=2, order=10)  
+    #   │   └── Grandchild (pk=4, order=5)
+    #   └── Child B (pk=3, order=20)
+
+    # For the Grandchild node:
+    grandchild = Node.objects.with_tree_fields().get(pk=4)
+    
+    # tree_path shows the route through primary keys: Root -> Child A -> Grandchild
+    assert grandchild.tree_path == [1, 2, 4]  # [root.pk, child_a.pk, grandchild.pk]
+    
+    # tree_ordering shows ordering values at each level: Root's order, Child A's order, Grandchild's order
+    assert grandchild.tree_ordering == [0, 10, 5]  # [root.order, child_a.order, grandchild.order]
+
+**Important note for UUID primary keys**: When using UUID primary keys without
```

Comment:
This doesn't just apply for UUID primary keys, but for all models without a sibling ordering field.